Minor proof-reading copy updates to MASVS content. (#369)
OWASP/owasp-masvs: Document/0x06-V1-Architecture_design_and_threat_modelling_requireme.md

| # | MSTG-ID | Description | L1 | L2 |
| --- | --- | --- | --- | --- |
| **1.1** | MSTG‑ARCH‑1 | All app components are identified and known to be needed. | ✓ | ✓ |

> | **1.2** | MSTG‑ARCH‑2 | Security controls are never enforced only on the client side, but also on the respective remote endpoints. | ✓ | ✓ |

Reviewer: After discussing internally, we would suggest the following:

```suggestion
| **1.2** | MSTG‑ARCH‑2 | Security controls are never enforced only on the client side, but on the respective endpoints as well. | ✓ | ✓ |
```

If you agree, please commit to the PR then we will merge.

Author: Sounds reasonable, I've updated the PR with the suggested tweak.

Reviewer: Perfect, thanks! Next time you can just use the "Comment Suggestion" button, in my comment above. It's a new feature that Github introduced last year. Then it's much easier for the creator of the PR to update it.
